[2025-04-23T14:40:54.517+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-23T14:40:55.498+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T14:40:46.662414+00:00 [queued]>
[2025-04-23T14:40:55.638+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T14:40:46.662414+00:00 [queued]>
[2025-04-23T14:40:55.667+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-23T14:40:55.997+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): LandingToBronze> on 2025-04-23 14:40:46.662414+00:00
[2025-04-23T14:40:56.047+0000] {standard_task_runner.py:63} INFO - Started process 480 to run task
[2025-04-23T14:40:56.077+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'final_project_BBDL', 'LandingToBronze', 'manual__2025-04-23T14:40:46.662414+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/project_solution.py', '--cfg-path', '/tmp/tmpm8sjh0ss']
[2025-04-23T14:40:56.097+0000] {standard_task_runner.py:91} INFO - Job 2: Subtask LandingToBronze
[2025-04-23T14:40:57.067+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.10/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-23T14:40:58.308+0000] {task_command.py:426} INFO - Running <TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T14:40:46.662414+00:00 [running]> on host 89b4ca2e2289
[2025-04-23T14:41:01.658+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='final_project_BBDL' AIRFLOW_CTX_TASK_ID='LandingToBronze' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T14:40:46.662414+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T14:40:46.662414+00:00'
[2025-04-23T14:41:01.678+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-23T14:41:02.208+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-23T14:41:02.217+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/landing_to_bronze.py", line 6, in main
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2025-04-23T14:41:02.768+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=final_project_BBDL, task_id=LandingToBronze, run_id=manual__2025-04-23T14:40:46.662414+00:00, execution_date=20250423T144046, start_date=20250423T144055, end_date=20250423T144102
[2025-04-23T14:41:03.107+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 2 for task LandingToBronze (No module named 'findspark'; 480)
[2025-04-23T14:41:03.257+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-04-23T14:41:03.847+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
